### ISSUE TITLE
Browser tests are not running under CI

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#3971]
### Share your bug report, feature request, or comment.

Running fine on my machine, however. This started happening out of nowhere.

```
<w> [webpack.cache.PackFileCacheStrategy] Caching failed for pack: Error: Unable to snapshot resolve dependencies
```

There's always something with a Javascript project that you're fixing.

**Comment by mahmed2000** [comment]
Probably related to the node module cache. It started 2 days ago, and the `Linux-build-cache-node-modules-browser-tests-76450d2...`  cache matches that under Actions -> Management -> Caches. Probably also why its only the browser tests failing.

![Image](https://github.com/user-attachments/assets/76d8b022-99d6-409b-81d6-1a61aec7fc65)

Is there a way to get that cache locally to test a build on? Though the fix might just be as simple as deleting it and letting it rebuild automatically.

**Comment by gabek** [comment]
I think this is two different types of caches. I'm not sure what the webpack cache is, or what's wrong with it. Just for kicks I cleared the NPM cache, but that didn't fix anything. It's probably some JavaScript dependency issue. I'll do some bisecting and see what's up.

**Comment by gabek** [comment]
I'm able to reproduce this locally using Node `v23.0.0`, but not previous versions.

**Comment by gabek** [comment]
Fixed in 742df5322f83bb3fed7d95d62334b93b847ac05e
